src/DyslexiaFont.ts

```diff
@@ -0,0 +1,32 @@
+import BookFont from "./BookFont";
+import * as HTMLUtilities from "./HTMLUtilities";
+
+export default class DyslexiaFont implements BookFont {
+  public readonly name = "dyslexia-friendly";
+  public readonly label = "Dyslexia Friendly";
+
+  public bookElement: HTMLIFrameElement;
+
+  public start(): void {
+    const rootFrame = HTMLUtilities.findRequiredIframeElement(this.bookElement.contentDocument, "html") as HTMLHtmlElement;
+
+    HTMLUtilities.setAttr(rootFrame, "data-viewer-font", "dyslexia-friendly");
+    HTMLUtilities.createStylesheet(rootFrame, "dyslexia-font-stylesheet", 
+    `@font-face {
```

Comment:
yes, this is the way to load a new font inside this iframe.   No, I haven't figured out how to do this via a css file.

Reply:
lol